pyproject.toml

```diff
@@ -25,10 +25,16 @@ invalid-assignment = "ignore"  # mypy is more permissive with Any assignments
 call-non-callable = "ignore"   # mypy allows more dynamic method calls
 index-out-of-bounds = "ignore" # mypy is more permissive with tuple indexing
 unresolved-attribute = "ignore" # mypy is more permissive with module attributes
-possibly-unbound-attribute = "ignore" # mypy doesn't warn about this as much
+possibly-unbound-variable = "ignore" # mypy doesn't warn about this as much
 redundant-cast = "ignore"      # mypy doesn't warn about redundant casts
```

Comment:
you had also applied these changes in another PR. should we add some comment in either PR about whhy and what happened?